### PR TITLE
Translated some missing stuff to spanish

### PR DESCRIPTION
**Pull request by ThePythonGuy3**
Some things were untranslated (i suppose this is the right branch to make this type of Pull Request)

### CODE REVIEW

**Comment by quentinguidee** [comment]
Hi! Thank you for this PR! Indeed, it's the right branch 👍

File: apps/home/base.es.i18n [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Apps = "Aplicaciones"
-AppsCapital = "OMEGA"
-ForbidenAppInExamMode1 = "This application is"
-ForbidenAppInExamMode2 = "forbidden in exam mode"
+AppsCapital = "QUÉDATE EN CASA(GA)"
```
Comment (quentinguidee) [inline apps/home/base.es.i18n:2]:
```suggestion
AppsCapital = "OMEGA"
```

This should stay "Omega" for Omega 1.21 ^^

**Review by quentinguidee — CHANGES_REQUESTED** [review]

**Comment by ThePythonGuy3** [comment]
done, thank you for pointing it out

**Comment by quentinguidee** [comment]
Thanks for the changes, and thank you for this PR! Should appear in Omega 1.21.0+ 👌

**Pull request merged**